stan/math/rev/fun.hpp

```diff
@@ -3,11 +3,6 @@
 
 #include <stan/math/prim/fun/Eigen.hpp>
 
-#include <stan/math/prim/err.hpp>
-#include <stan/math/prim/fun.hpp>
-#include <stan/math/prim/functor.hpp>
-#include <stan/math/prim/meta.hpp>
```

Comment:
Why are these removed? I mean they should probably be included after all the rev stuff, but I think they should be in this file.

Reply:
Ack these were removed while trying to fix the includes. Added them back. 

I'm going to make a separate issue on fixing the headers, I think there's some weird stuff going on